.github/CONTRIBUTING.md

```diff
@@ -86,6 +86,17 @@ Also mind that CoffeeScript is production dependency (not dev dependency),
 because it's needed not only for compiling Dredd package before uploading
 to npm, but also for running user-provided hooks written in CoffeeScript.
 
+### Compiled vs pure JavaScript
+
+Dredd uses [Drafter][] for parsing [API Blueprint][] documents. Drafter is written in C++11 and needs to be compiled during installation. Since that can be problematic for some environments and leads to a lot of troubleshooting, there's also pure JavaScript version of the parser, [drafter.js][]. While drafter.js is fully equivalent, it can have slower performance. That's why there's [drafter-npm][] package, which first tries to compile the C++11 version of the parser and if it's unsuccessful, uses the JavaScript equivalent.
+
+Dredd depends on the [drafter-npm][] package. That's why you can see `node-gyp` errors and failures during Dredd installation, although after the installation is done, Dredd seems to normally work and correctly parses API Blueprint documents. Usual problems leading to the JavaScript version of the parser being used as fallback:
```

Comment:
`That's the reason why you can see `node-gyp` errors and failures...`
`even though installation ended correctly and Dredd...` 

Last sentence doesn't make sense for me - are you trying to fix this issues? Is it necessary if it seems to be working? I would appreciate more explanation on the goal you want to achieve with the following list.